app/src/main/java/chat/rocket/android/main/presentation/MainPresenter.kt

```diff
@@ -354,6 +360,42 @@ class MainPresenter @Inject constructor(
         }
     }
 
+    fun shareViaApp(context: Context){
+        launch {
+            //get serverUrl and username
+            val server = serverInteractor.get()!!
+            val account = getAccountInteractor.get(server)!!
+            val userName = account.userName
+
+            FirebaseDynamicLinks.getInstance().createDynamicLink()
+                .setLink(Uri.parse("$server/direct/$userName"))
+                .setDomainUriPrefix("https://dylinktesting.page.link")
```

Comment:
Can you make this configurable instead of hard coding the domain here, maybe in the Constants.kt file?